src/utils/point_locator_list.C

```diff
@@ -192,6 +192,35 @@ const Elem * PointLocatorList::operator() (const Point & p,
 }
 
 
+void PointLocatorList::operator() (const Point & p,
```

Comment:
PointLocatorList is officially deprecated, so I wouldn't spend too much time on this... or simply leave it as `libmesh_unimplemented()`.